sources/engine/Stride.Engine/Engine/Splines/Models/Spline.cs

```diff
@@ -0,0 +1,291 @@
+//// Copyright (c) Stride contributors (https://Stride.com)
+//// Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.Collections.Generic;
+using Stride.Core;
+using Stride.Core.Mathematics;
+
+namespace Stride.Engine.Splines.Models
+{
+    [DataContract]
```

Comment:
Why is this class serializable? In the component the property with this class is ignored.